runtime/src/bank/serde_snapshot.rs

```diff
@@ -535,7 +535,7 @@ mod tests {
         #[cfg_attr(
             feature = "frozen-abi",
             derive(AbiExample),
-            frozen_abi(digest = "8hwm4YsQXJWGZdp762SkJnDok29LXKwFtmW9oQ2KSzrN")
+            frozen_abi(digest = "BzZMy3Mia1DHMaQGwSyPPChUWSw8MpHsZEwGWGDyMJFS")
```

Comment:
what's fqn?

Reply:
probably, yes. I don't know how the digest is computed, but since `BankHashStats` is moved in a new module and it derives AbiExample, I guess it does change the digest.

Reply:
Where can I find abi export files or how to make them?

Reply:
I think you need to set `SOLANA_ABI_DUMP_DIR` env-var and rerun the test.

Reply:
compared the diffs before and after this change

```
$ ll abi_dir_*
abi_dir_after:
total 248K
-rw-rw-r-- 1 dmakarov  17K Oct  7 12:48 bank__BankSlotDelta_frozen_abi__test_abi_digest_BswQL6n7kKwgHFKcwMCQcrWjt8h59Vh6KkNb75iaqG2B
-rw-rw-r-- 1 dmakarov 225K Oct  7 12:48 bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_7xkyjhBmj1xk3ykcbufPCnBKKkcpQ3AjKFUmH1r8MRnu

abi_dir_before:
total 248K
-rw-rw-r-- 1 dmakarov  17K Oct  7 12:49 bank__BankSlotDelta_frozen_abi__test_abi_digest_BswQL6n7kKwgHFKcwMCQcrWjt8h59Vh6KkNb75iaqG2B
-rw-rw-r-- 1 dmakarov 225K Oct  7 12:49 bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_8hwm4YsQXJWGZdp762SkJnDok29LXKwFtmW9oQ2KSzrN
2165:~/work/git/agave:master % u=$ diff -rq abi_dir_before abi_dir_after 
Only in abi_dir_after: bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_7xkyjhBmj1xk3ykcbufPCnBKKkcpQ3AjKFUmH1r8MRnu
Only in abi_dir_before: bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_8hwm4YsQXJWGZdp762SkJnDok29LXKwFtmW9oQ2KSzrN

$ diff abi_dir_before/bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_8hwm4YsQXJWGZdp762SkJnDok29LXKwFtmW9oQ2KSzrN abi_dir_after/bank__serde_snapshot__tests__test_bank_serialize__BankAbiTestWrapper_frozen_abi__test_abi_digest_7xkyjhBmj1xk3ykcbufPCnBKKkcpQ3AjKFUmH1r8MRnu 
1778c1778
<                             field stats: solana_accounts_db::accounts_db::BankHashStats
---
>                             field stats: solana_accounts_db::accounts_db::stats::BankHashStats

```

@brooksprumo what results do you expect?